src/main/java/com/google/codeu/data/Laptop.java

```diff
@@ -3,22 +3,31 @@
 import java.util.UUID;
 
 public class Laptop {
+	//all lower case
 	  private UUID id;
 	  private String brand;
 	  private String color;
 	  private String os;
 	  private int size;
 	  private double price;
 	  
+	  public Laptop() {
+		  id=UUID.randomUUID();
+		  brand=null;
+		  color=null;
+		  os=null;
+		  size=0;
+		  price=0;
+	  }
 	  public Laptop(String brand, String color, String os, int size, double price)  {
-		    this(UUID.randomUUID(), brand, color, os, size, price);
+		    this(UUID.randomUUID(), brand.toLowerCase(), color.toLowerCase(), os.toLowerCase(), size, price);
 		  }
 
 	  public Laptop(UUID id, String brand, String color, String os, int size, double price) {
 		this.id = id;
-	    this.brand = brand;
-	    this.color = color;
-	    this.os = os;
+	    this.brand = brand.toLowerCase();
```

Comment:
Is it ever possible for any of these values to be null? What should happen in that case?